### PR TITLE
Changing delivery status on orders as well as customer login page

### PR DESCRIPTION
**Pull request by ooj2899**

### CODE REVIEW

**Comment by santoniriccardo** [comment]
Seems like there are conflicts given new push. If these could be resolved I will accept!

**Comment by ooj2899** [comment]
resolved conflicts

File: frontend/src/components/Customer/CustomerSignUp.tsx [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+import ImageList from '@mui/material/ImageList';
+import ImageListItem from '@mui/material/ImageListItem';
+import Container from '@mui/material/Container';
+import { createTheme, makeStyles, ThemeProvider } from '@mui/material/styles';
+import DronutIcon from '../../assets/dronut.png'
+import { ElectricScooterOutlined, Label } from '@mui/icons-material';
+import LocationInterface from './Location';
+
+
+const itemData = [
```
Comment (santoniriccardo) [inline frontend/src/components/Customer/CustomerSignUp.tsx:23]: Can we use donutImages.tsx here?

File: frontend/src/components/Customer/CustomerSignUp.tsx [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+    phone_number: string, 
+    password: string,
+    address: Array<LocationInterface>
+}
+
+
+
+export function CustomerSignUp() {
+    
+    const [customerId, setCustomerId] = React.useState(0);
```
Comment (santoniriccardo) [inline frontend/src/components/Customer/CustomerSignUp.tsx:160]: I would recommend introducing a CustomerInterface similarly to OrderInterface or DonutInterface, and useState on the one object. Then this is passed through to backend API, and updated in form.

File: frontend/src/components/Employee/Orders.tsx [hunk 3]
```diff
@@ -0,0 +1,219 @@
+import SendIcon from '@mui/icons-material/Send';
```
Comment (santoniriccardo) [inline frontend/src/components/Employee/Orders.tsx:1]: We shouldn't have this file anymore after my most recent changes. Is there a reason for it being included?

File: frontend/src/components/Home/Login.tsx [hunk 4]
```diff
@@ -0,0 +1,354 @@
...
+import ImageList from '@mui/material/ImageList';
+import ImageListItem from '@mui/material/ImageListItem';
+import Container from '@mui/material/Container';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import DronutIcon from '../../assets/dronut.png'
+import { Label } from '@mui/icons-material';
+import Alert from '@mui/material/Alert';
+
+
+const itemData = [
```
Comment (santoniriccardo) [inline frontend/src/components/Home/Login.tsx:23]: Similarly, can we not use donutImages.tsx?

File: frontend/src/components/Home/Login.tsx [hunk 5]
```diff
@@ -0,0 +1,354 @@
...
+  async function LoginCustomer() {
+    try{
+      const response = await fetch('/customers').then((res) => res.json());
+      console.log("response", response);
+      window.alert("get here");
+    }catch (e){
+      console.log(e)
+    }
+  }
+//   const [customers, setCustomers] = React.useState<Array<customerInfo>>([]);
```
Comment (santoniriccardo) [inline frontend/src/components/Home/Login.tsx:180]: nit: lets remove comments/redundant code.

**Review by santoniriccardo — CHANGES_REQUESTED** [review]

File: frontend/src/components/Customer/CustomerSignUp.tsx (hunk 1 above)
Comment (ooj2899) [reply frontend/src/components/Customer/CustomerSignUp.tsx:23]: yeah

File: frontend/src/components/Employee/Orders.tsx (hunk 3 above)
Comment (ooj2899) [reply frontend/src/components/Employee/Orders.tsx:1]: not sure I still see it in my order.tsx

**Pull request merged**